### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by Tequilac**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,35 @@
 # To już jest projekt...
+
+
+## Uruchamianie
+Program uruchamiamy poprzez odpalenie pythonowego modułu **src/core** - reszta komponentów jest uruchamiana razem z nim.
+Wymagana wersja Pythona >= 3.9.
```
Comment (Kamilbur) [inline README.md]: That is not how it works XD sorry. It is just some magic script running it all.

**Review by Kamilbur — COMMENTED** [review]
Just that one thing needs a fix.
image
image

**Comment by Dawid126** [comment]
image

**Review by Kamilbur — APPROVED** [review]
image

**Pull request merged**